### PR TITLE
[6.5][NET] Move NUI and WCircularUI to User interface section.

### PR DESCRIPTION
**Pull request by mihashco**
This commit moves NUI and WCircularUI to one user interface section.
User interface is common and important topic. This change makes it
easier to search for it in the documentation tree.

Modified files: 
docs/applicaiton/toc_all.md
docs/application/dotnet/guides/index.md

### CODE REVIEW

**Comment by annie-samsung** [comment]
@sagarkedari this is 1491 PR content re-raised, could you check that all our points discussed in 1491 is addressed.

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@mihashco, I have added my review feedback comments. Kindly check and incorporate the changes.

File: docs/application/toc_all.md [hunk 1]
```diff
@@ -77,6 +77,62 @@
 #### [Message Port](/application/dotnet/guides/app-management/message-port.md)
 #### [Data Control](/application/dotnet/guides/app-management/data-control.md)
 
+### User interface
```
[the remark this answers is not in the document]
Comment (mihashco) [reply docs/application/toc_all.md:80]: @annie-samsung If you want I can move Display Web Content to guides in separated PR. Please let me know to confirm that change.

File: docs/application/dotnet/guides/index.md [hunk 2]
```diff
@@ -281,13 +271,11 @@ Tizen.Wearable.CircularUI supports Tizen wearable-specific user interfaces and T
     <div class="col col-md-3">
         <div class="card card-infra card-list h-100">
             <div class="card-body">
-                <p class="h3 card-title">NUI</p>
+                <p class="h3 card-title">User interface</p>
```
[the remark this answers is not in the document]
Comment (mihashco) [reply docs/application/dotnet/guides/index.md:274]: Not all of them. Display Web Content is at the end of the list.

[the remark this answers is not in the document]
Comment (sagarkedari) [reply docs/application/dotnet/guides/index.md]: @mihashco, Yes. After @annie-samsung confirmation on raising new PR for relocating "display web content" in the TOC file,  I thought of making change in this topic/file as well and include both files in the new PR.

**Review by sagarkedari — CHANGES_REQUESTED** [review]
@mihashco, I have added my review feedback comments. Kindly check and incorporate the changes.

File: docs/application/dotnet/guides/index.md [hunk 3]
```diff
@@ -400,15 +387,16 @@ Tizen.Wearable.CircularUI supports Tizen wearable-specific user interfaces and T
     <div class="col col-md-3">
         <div class="card card-infra card-list h-100">
             <div class="card-body">
-                <p class="h3 card-title">Display Web Content</p>
+                <p class="h3 card-title">User interface</p>
```
[the remark this answers is not in the document]
Comment (mihashco) [reply docs/application/dotnet/guides/index.md:390]: This commit don't change gaps. It only adds one box. Looks of the index page was introduced in other commit.

**Review by sagarkedari — APPROVED** [review]
@annie-samsung, This PR is approved from my end. Regarding following two comments, I have made a note to check with concerned developer for these issue resolution.

https://github.com/Samsung/tizen-docs/pull/1511#discussion_r717259627
https://github.com/Samsung/tizen-docs/pull/1511#discussion_r717301696

**Comment by annie-samsung** [comment]
Merging this PR as it is long pending and reviewed by Sagar twice now

**Pull request merged**